src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -231,6 +231,22 @@ internal void ClearPointerState()
 			ClearDragOver();
 		}
 
+#if UNO_HAS_MANAGED_POINTERS
+		internal void AbortGestureRecognizer()
+		{
+			var element = this;
+			while (element is not null)
+			{
+				if (element.IsGestureRecognizerCreated)
+				{
+					element.GestureRecognizer.CompleteGesture();
```

Comment:
Please use the existing (public) [`CompleteGesture`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/UIElement.Pointers.cs#L580) instead. 

I would then rename your method to something like `CompleteGesturesOnTree` (or anything better than that) to match the current API name and indicate that it's not only on the current element. (Please also move your method next to the existing `CompleteGesture`.)